Silicon/MeteorlakePkg/Library/BootGuardLibCBnT/BootGuardLibCBnT.inf

```diff
@@ -0,0 +1,51 @@
+/** @file
+  Component description file for Boot Guard Library.
+
+  Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
```

Comment:
Need use BSD plus license. Please check if you could use common lib Silicon\CommonSocPkg\Library\BootGuardLibCBnT

Reply:
patch submitted with removed duplicate lib